docs/src/main/sphinx/release/release-365.md

```diff
@@ -0,0 +1,132 @@
+# Release 365 (XX Dec 2021)
+
+## General
+
+* Add support for [`TRUNCATE TABLE`](/sql/truncate). ({issue}`8921`)
+* Add support for time travel queries. ({issue}`8773`)
+* Add [`contains`](ip_address_contains) function to check whether a CIDR contains an IP address. ({issue}`9654`)
+* Report connector metrics in `EXPLAIN ANALYZE VERBOSE`. ({issue}`9858`)
+* Report operator input distribution in ``EXPLAIN ANALYZE VERBOSE``. ({issue}`10133`)
+* Allow executing ``INSERT`` or ``DELETE`` statements on tables restricted with a row filter. ({issue}`8856`)
+* Remove `owner` column from the `metadata.materialized_views` table. ({issue}`9961`) 
+* Improve performance of inequality joins. ({issue}`9307`)
+* Improve performance of joins involving a small table on one side. ({issue}`9851`)
+* Improve CPU utilization by adjusting ``task.concurrency`` automatically based on the number of physical cores. ({issue}`10088`)
+* Make query final statistics more accurate. ({issue}`9888`, {issue}`9913`)
+* Improve query planning performance for queries containing large `IN` predicates. ({issue}`9874`)
+* Fix incorrect results when casting `bigint` values to `varchar(n)` type. ({issue}`552`)
+* Fix query failure when `PREPARE` statement is used with `DROP` or `INSERT` when table or schema name 
```

Comment:
nit: `.` at the end

Reply:
Also, how about the following (also refer to issue instead of pr)

```
Fix query failure for the `PREPARE` statement, when it is used with `DROP` or `INSERT` 
statements, and the table or schema name contains special characters. ({issue}`9822`)
```

Reply:
I adjusted the phrasing a bit based on your suggestion.